openhcl/underhill_core/src/worker.rs

```diff
@@ -2672,13 +2676,41 @@ async fn new_underhill_vm(
             let builder = vmbus_client_hcl::vmbus_client_builder(relay_driver)
                 .context("failed to create synic client and message source")?;
 
+            let mut client = builder.build(tp);
+            let connection = if let Some(state) = servicing_state.vmbus_client.flatten() {
+                client.restore(state).await?
+            } else {
+                None
+            };
+            client.start();
+            let connection = if let Some(c) = connection {
+                c
+            } else {
+                const VMBUS_RELAY_CLIENT_ID: Guid =
```

Comment:
where is this constant coming from? is it some known-shared magic constant? an arbitrary guid?

Reply:
Arbitrary GUID so that we can understand what's guest implementation we're dealing with from the host.

Reply:
Commented